lib/inferno/utils/evaluator_util.rb

```diff
@@ -0,0 +1,137 @@
+module Inferno
+  module Utils
+    module EvaluatorUtil
```

Comment:
It's not clear to me that this belongs here rather than in with the evaluator.

Reply:
I guess it's the question of how we envision utilities for the evaluator can be in general use by Inferno. @dehall Do you have any idea?

Reply:
It all makes sense. I will move this under `dsl/fhir_evaluation` Since it's (and will) being used by multiple rules in a shared manner, I think this should be still a separate class as a file, but I will make it simplest as possible.

Reply:
Done https://github.com/inferno-framework/inferno-core/pull/583/commits/85a2bd47e8264f679551c9a16a4a0643f5c3726e